rbac.yaml

```diff
@@ -12,6 +12,16 @@ rules:
   - list
   - update
   - watch
+- apiGroups: [""] # "" indicates the core/v1 API group
+  resources:
+  - events
+  verbs:
+  - create
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
Do you actually `watch` for events?

Reply:
No needed, I removed the path,watch,list